### DISCUSSION TITLE
Add random noise to adjacency matrix

### DISCUSSION THREAD

**Discussion by shrimonmuke0202** [pyg-team/pytorch_geometric#2585] [Q&A]
How to add random trainable Gaussian noise to the adjacency matrix of a graph?

**Comment by rusty1s** [comment]
See https://github.com/rusty1s/pytorch_geometric/discussions/2493#discussioncomment-673292

**Comment by shrimonmuke0202** [comment]
adj = adj + 0.01 * torch.rand_like(adj,requires_grad=True)
using the above code can I add random noise ?

**Reply by rusty1s** [reply]
Yes.
